lib/PanelTestProvider.jsm

```diff
@@ -5,20 +5,39 @@
 
 const MESSAGES = () => ([
   {
-    "id": "SIMPLE_FXA_BOOKMARK_TEST_1",
+    "id": "SIMPLE_FXA_BOOKMARK_TEST_FLUENT",
     "template": "fxa_bookmark_panel",
     "content": {
-      "title": "cfr-doorhanger-bookmark-fxa-header",
-      "text": "cfr-doorhanger-bookmark-fxa-body",
-      "cta": "cfr-doorhanger-bookmark-fxa-link-text",
+      "title": {"string_id": "cfr-doorhanger-bookmark-fxa-header"},
+      "text": {"string_id": "cfr-doorhanger-bookmark-fxa-body"},
+      "cta": {"string_id": "cfr-doorhanger-bookmark-fxa-link-text"},
       "color": "white",
       "background_color_1": "#7d31ae",
       "background_color_2": "#5033be",
       "info_icon": {
-        "tooltiptext": "cfr-doorhanger-bookmark-fxa-info-icon-tooltip",
+        "tooltiptext": {"string_id": "cfr-doorhanger-bookmark-fxa-info-icon-tooltip"},
       },
       "close_button": {
-        "tooltiptext": "cfr-doorhanger-bookmark-fxa-close-btn-tooltip",
+        "tooltiptext": {"string_id": "cfr-doorhanger-bookmark-fxa-close-btn-tooltip"},
+      },
+    },
+    "trigger": {"id": "bookmark-panel"},
+  },
+  {
+    "id": "SIMPLE_FXA_BOOKMARK_TEST_NON_FLUENT",
```

Comment:
Remind me, what makes the test bookmark provider content show up v.s. not show up?

Reply:
The provider [sets targeting to `true`](https://github.com/mozilla/activity-stream/blob/8d5dc9939f14bf819b5646f22565bd6bf760c688/lib/PanelTestProvider.jsm#L31) so that the message always shows up. This was needed because we didn't have any messages in the `cfr-fxa` bucket.
Last week we landed the message in the production bucket so I can turn this off to only show up if you click `Show` in the admin page. I should be able to attach the message to the panel of the currently selected window even if we don't have a way to trigger the panel to drop down.